leaf-ui/rappid-extensions/AnalysisInspector.js

```diff
@@ -77,6 +77,7 @@ var AnalysisInspector = Backbone.View.extend({
 		// These functions are used to communicate between analysisInspector and Main.js
 		this._analysisFunctions = analysisFunctions;
 		this.$el.html(_.template(this.template)());
+		$('head').append('<script src="./scripts/js-objects/analysis.js"></script>');
```

Comment:
Added back this line so analysis window pops up. Removed before because this line was crashing the code and I couldn't even test the changes.